config/config.go

```diff
@@ -46,8 +49,9 @@ type (
 		HttpServer HttpServerConfig `yaml:"httpServer"`
 		// Rpc is the config for rpc calls
 		Rpc RpcConfig `yaml:"rpc"`
-		// AsyncServiceAddresses are the addresses for API service to call AsyncServices' internal APIs
-		AsyncServiceAddresses []string `yaml:"asyncServiceAddresses"`
+		// AsyncServiceAddress is the address for API service to call the AsyncService's internal APIs
+		// It's required in the standalone mode, but not needed in the cluster mode
```

Comment:
This makes me feel like we probably can deprecate standalone mode, or in other words, it can become automatic somehow in a smart way

Reply:
It's doable, it means the standalone mode will be equal to the cluster mode with only 1 async server, and we need to use memberlist anyway.